cmd/utils/flags.go

```diff
@@ -1883,7 +1888,7 @@ func SplitTagsFlag(tagsFlag string) map[string]string {
 }
 
 // MakeChainDatabase open an LevelDB using the flags passed to the client and will hard crash if it fails.
-func MakeChainDatabase(ctx *cli.Context, stack *node.Node, readonly bool) ethdb.Database {
+func MakeChainDatabase(ctx *cli.Context, stack *node.Node, readonly, disablFreeze bool) ethdb.Database {
```

Comment:
typo `disablFreeze` -> disableFreeze

Reply:
Updated. I'm aware of this and my intention was to simplify the variable, but anyway I updated.